pkgs/build-support/setup-hooks/install-shell-files.sh

```diff
@@ -122,6 +122,17 @@ installShellCompletion() {
                     return 1
                 }
                 continue;;
+            --size)
+                size=$1
+                shift || {
+                    echo 'installShellCompletion: error: --size flag expected an argument' >&2
+                    return 1
+                }
+                continue;;
+            --size=*)
+                # treat `--size=foo` the same as `--size foo`
+                size=${arg#--size=}
+                continue;;
```

Comment:
`--size` looks like this is going to do something relevant with this flag. I would suggest changing this to `--size-check` or something to make it more clear that this is only a sanity check for files.

Reply:
But instead of this, could we have a smarter check instead? Something that validates that the `manpage` is in the correct format for example? Could we run `troff` in some kinda of check mode and exit if it fails?

Reply:
> `--size` looks like this is going to do something relevant with this flag. I would suggest changing this to `--size-check` or something to make it more clear that this is only a sanity check for files.

Fixed. I liked the `--size-check` option name.